projects/plugins/jetpack/extensions/blocks/conversation/utils.js

```diff
@@ -14,6 +15,11 @@ export function getParticipantByLabel ( participants, participantLabel ) {
 	return part?.length ? part[ 0 ] : null;
 }
 
-export function cleanFormatStyle( html ) {
-	return getTextContent( create( { html } ) )?.trim();
+export function getPlainText( html, escape = false ) {
+	const text = getTextContent( create( { html } ) )?.trim();
+	if ( ! escape ) {
+		return text;
+	}
+
+	return escapeHTML( text );
```

Comment:
Is this actually needed? By this point, do we ever have any HTML that needs escaping?

Reply:
it escapes the HTML when adding/updating speaker in the Conversation block.

canvas | sidebar | Conversation block
------|------|------
<img width="460" src="https://user-images.githubusercontent.com/77539/107696623-f5365080-6c90-11eb-90c0-8844a5ed4ff3.png"> | <img width="460" src="https://user-images.githubusercontent.com/77539/107696637-f8c9d780-6c90-11eb-91ce-7a02ab8e2933.png"> | <img width="460" alt="Screen Shot 2021-02-11 at 5 45 56 PM" src="https://user-images.githubusercontent.com/77539/107696680-08e1b700-6c91-11eb-88bb-484771ef63f5.png">

Reply:
I see it now: running a speaker name through `getTextContent( create( { html } ) )?.trim()` will actually restore already escaped html... so we need to escape it again.

Seems like there's still something not quite right about this approach, but it works for now.